src/app/app.coffee

```diff
@@ -4,18 +4,27 @@
 # Doubtfire is modularised into many modules, as indicated by the directory
 # tree inside app/
 #
-#
+
+# Require the CSS file explicitly (or it could be defined as an entry-point too).
+require('../assets/styles/app.scss')
+
+# angular-markdown-filter expects window.showdown to exist
+window.showdown = require('showdown')
+# angular-ui-codemirror expects window.CodeMirror
+window.CodeMirror = require('codemirror')
```

Comment:
To me it clutters up the app file unnecessarily. Since it's vendor-specific I think it should be best kept next to the angular include:

```coffee
# angular-ui-codemirror expects window.CodeMirror
window.CodeMirror = require('codemirror')
uiCodemirror = require('angular-ui-codemirror')
```

We don't use raw CodeMirror anywhere else, so why would it be promoted up into `app.coffee`?

:+1: on moving it out of config. Let's rename it `vendor` and include it before the module definition. Maybe we something similar for styles also.